fee-monitor/src/index.ts

```diff
@@ -33,13 +33,42 @@ async function getNextBlockNumber(current: number) {
     }
 }
 
+async function getCurrentTermStartBlockNumber(blockNumber: number): Promise<number | null> {
+    return new Promise((resolve, reject) => {
+        sdk.rpc
+            .sendRpcRequest("chain_getTermMetadata", [blockNumber])
+            .then(result => {
+                if (result === null) {
+                    resolve(null);
+                }
+                const [prevTermLastBlockNumber, currentTermId] = result;
```

Comment:
Check that it is an array before destructuring it too.

Reply:
Thank you for patching